dracut/30ignition/module-setup.sh

```diff
@@ -57,24 +57,16 @@ install() {
 #       "$systemdsystemunitdir/coreos-static-network.service"
 }
 
-has_builtin_fw_cfg() {
-    # this is like check_kernel_config() but it specifically checks for `y` and
-    # also checks the OSTree-specific kernel location
+installkernel() {
+    # We definitely need this one in the initrd to support Ignition cfgs on qemu
     for path in /boot/config-$kernel \
                 /usr/lib/modules/$kernel/config \
                 /usr/lib/ostree-boot/config-$kernel; do
         if test -f $path; then
-            rc=0
-            grep -q CONFIG_FW_CFG_SYSFS=y $path || rc=$?
-            return $rc
+            if grep -q CONFIG_FW_CFG_SYSFS=y $path; then
```

Comment:
if `CONFIG_FW_CFG_SYSFS=y` exists then the functionality is built in to the kernel meaning no module is needed, right?

Reply:
It includes the module if it has been enabled in the kernel configuration. If I understand that correctly if it is part of the current running or installed kernel.

Reply:
Disregard the last comment...